homing_missiles.py

```diff
@@ -89,6 +102,9 @@ def draw(self, screen):
         self.draw_entities(screen)
         self.draw_target_tags(screen)
         self.targeting_terminal_.draw_terminal(screen)
+        
+        self.draw_friendly_projectiles(screen)
+        self.draw_targets(screen)
```

Comment:
So the plan here is just to draw over them? I'd actually prefer that rather than a whole change of color, they have a border around them.